components/EventsList.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useRef, useState } from "react"
+import { ListRenderItemInfo, View } from "react-native"
+import BottomSheet, {
+  BottomSheetFlatList,
+  BottomSheetModal,
+  BottomSheetModalProvider
+} from "@gorhom/bottom-sheet"
+import { Event } from "@lib/events/Event"
+import EventItem from "@components/EventItem"
+import NearbyActivities from "./headerComponents/NearbyActivities"
+import { Events, GraphQLEventItems } from "@lib/events/Events"
+
+let eventItems: Events
+
+const EventsList = () => {
+  eventItems = new GraphQLEventItems()
+  const ids = Array.from(new Array(10), (_, i) => String(i))
+  const events = eventItems.eventsWithIds(ids)
+
+  // hooks
+  const sheetRef = useRef<BottomSheetModal>(null)
+
+  useEffect(() => {
```

Comment:
I haven’t seen .present() before, what does it do?

Reply:
.present() calls opens the modal. By default the modal is closed when the component is rendered, which means it doesn't appear on the screen at all. In order for the user to swipe it up and down the modal has to be open which is why I'm calling .present() when the component is first rendered.